### DISCUSSION TITLE
Unraid container, whats missing? Channels self delete, frustrated.

### DISCUSSION THREAD

**Discussion by AnterosOberon** [chrisbenincasa/tunarr#891] [Questions & Help]
So, I feel frustrated, I keep setting up this app, and using it, and it works great for a couple days until i stop baby sitting it, i install an update and then, again, everything is gone. I was looking at the unraid docker template and its so basic it must be missing some settings that are needed to invoke saved settings or a backup or something. 

This is what my template i am using looks like, what if anything is missing.. 
![firefox_5v8Wm5gNMU](https://github.com/user-attachments/assets/a7aa1fdb-bf95-45df-961d-c00d7db47dc6)

**Comment by chrisbenincasa** [comment]
It looks like the issue is that you may be using an old version of the unraid template. I see that it is mapping your data to `/tunarr/server/build/.tunarr` _within_ the container (this is the "Container Path" listed under the configuration data option). This was changed to the new default `/config/tunarr` in the template a very long time ago: https://github.com/nwithan8/unraid_templates/blob/main/templates/tunarr.xml#L80 (in this PR: https://github.com/nwithan8/unraid_templates/pull/30). I think this is definitely one of the problems

**Comment by chrisbenincasa** [comment]
Hey there - wondering if you were ever able to get this working
